router/server.go

```diff
@@ -67,9 +68,19 @@ func main() {
 	tcpIP := flag.String("tcpip", "", "tcp router listen ip")
 	tcpRangeStart := flag.Int("tcp-range-start", 3000, "tcp port range start")
 	tcpRangeEnd := flag.Int("tcp-range-end", 3500, "tcp port range end")
+	certFile := flag.String("tlscert", "", "TLS (SSL) cert file in pem format")
+	keyFile := flag.String("tlskey", "", "TLS (SSL) key file in pem format")
```

Comment:
Will that fallback work? The environment variable will contain the data, but the flag is supposed to be a file path. The flag names are fine, but the description should note that they are for the default certificate presented by the HTTP listener (not the API, which is also HTTP).